jax/numpy/lax_numpy.py

```diff
@@ -2610,6 +2610,26 @@ def polyval(p, x):
   return y
 
 
+def trim_zeros(a):
+  first = 0
+  for i in a:
+    if i != 0:
+      break
+    first = first + 1
+  return a[first:]
+
+@_wraps(np.polymul)
+def polymul(a1, a2, trim_leading_zeros=False):
```

Comment:
use `def polymul(a1, a2, *, trim_leading_zeros=False):` so that `trim_leading_zeros` is keyword-only.